compliance/solr/pom.xml

```diff
@@ -62,6 +66,11 @@
 			<version>${solr.version}</version>
 			<scope>test</scope>
 			<exclusions>
+				<!-- FIXME: Uncomment; Jetty 10.x requires Solr 9.x -->
+				<!--exclusion>
+					<groupId>javax.servlet</groupId>
+					<artifactId>javax.servlet-api</artifactId>
+				</exclusion -->
```

Comment:
Also not sure what this means for us.

Reply:
We must choose to have jakarta.* **or** javax.* dependencies. Never both. As there is a BOM for Jakarta dependencies, I would prefer to use that one.